{"sid": "6001480468f74a06ac290fa50a71bfb8", "uid": null, "qid": "b66ec0622aa64e7a91bb019d93110428", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [b66ec0622aa64e7a91bb019d93110428]\nTraceback (most recent call last):\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/action_dispatcher.py\", line 139, in process\n    output = self.workerContext.execution.execute(data, self.ray, self.state, update)\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/application_interface.py\", line 66, in execute\n    self.execution_function(model)\n  File \"/openfabric/app/main.py\", line 93, in execute\n    msg , img , obj , sessid  = agent.Exec(user_prompt)\n  File \"/openfabric/app/Agent/agent.py\", line 71, in Exec\n    text = self.process(response)\n  File \"/openfabric/app/Agent/agent.py\", line 114, in process\n    self.generate_image(parsed)\n  File \"/openfabric/app/Agent/agent.py\", line 95, in generate_image\n    self.image = Agent.generator.generate_image(image_description)\n  File \"/openfabric/app/main.py\", line 54, in generate_image\n    result = cls.stub.call(cls.app_ids[0], {'prompt':prompt}, 'super-user')\nIndexError: list index out of range\n", "created_at": "2025-06-28T12:53:03.126699"}], "status": "FAILED", "created_at": "2025-06-28T12:52:49.481908", "updated_at": "2025-06-28T12:53:03.126741", "finished": true}